src/pages/hiring-process/components/hiring-process-list/index.js

```diff
@@ -36,17 +36,7 @@ export const ProcessList = ({ processes, setHiringProcesses }) => {
     location.reload()
   }
 
-  const handleDelete = async (id) => {
-    try {
-      const answer = confirm(t('hiringProcess.delete'))
-      if (answer === false) return
-      client.delete(`/hiring_process/${id}`)
-      const newProcesses = processes.filter(process => process.id !== id)
-      setHiringProcesses(newProcesses)
-    } catch (error) {
-      console.error(error)
-    }
-  }
+  const isHiringProcessOpen = (status) => status === 'status-opened'
```

Comment:
outra coisa é que confirmei coma Ju Martins, e esse bloqueio deve ser feito somente quando o processo for fechado, enquanto ele estiver aberto deve permitir a edição.

entao tem q trocar pra `status-closed`